### PR TITLE
adding semáforo homework

### PR DESCRIPTION
**Pull request by ncrhodes**
Olá Bruno,

Não sei porque mas parece que também estou subindo o dever do "fashion blog"...Não sei se o meu github não estava atualizado ou que é que acontece. 

Nathan

### CODE REVIEW

**Comment by nsuitter** [comment]
@ncrhodes I don't see an assignment from you submitted on Moodle. Could you submit one please so we can assign you a grade? Thanks!

**Comment by ncrhodes** [comment]
Hey Nathan,

I appreciate your taking the time to point out that I had forgotten to
submit the link to the pull request on Moodle. I went ahead and did it now.
See you Monday.

Cheers,

Nathan

2015-03-20 20:37 GMT-07:00 nsuitter notifications@github.com:

File: Get_a_name/Deberdelsemáforo/index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	<title>Sinal brasileiro</title>
+
+	<link rel="stylesheet" type="text/css" href="main.css">
+
+</head>
+
+
+
+<body>
+<div id="controlPanel">
```
Comment (nsuitter) [inline Get_a_name/Deberdelsemáforo/index.html:13]: these div tags and child tags should be indented one level from the body tag

File: Get_a_name/Deberdelsemáforo/main.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  document.getElementById('stopLight').style.backgroundColor = "red";
+}
+
+function illuminateYellow() {
+  clearLights();
+  document.getElementById('slowLight').style.backgroundColor = "yellow";
+}
+
+function illuminateGreen() {
+	clearLights();
```
Comment (nsuitter) [inline Get_a_name/Deberdelsemáforo/main.js:16]: this indentation should be consistent with that of the other functions in this file

**Comment by nsuitter** [comment]
@ncrhodes nice work on this assignment
Other than a few indentation issue, it looks great. Nice solution to create a clearLights function to avoid repeating yourself in all of the other function.
:+1:

**Pull request merged**